corehq/apps/domain/static/domain/js/encoder.js

```diff
@@ -0,0 +1,21 @@
+function HexParsr(salt1, salt2) {
```

Comment:
Considering we don't need any of this JS to use anywhere else is it fine to just have it within a function and not use hqDefine?
Plus for hqDefine would need to include hqModules JS on this page as well.